### ISSUE TITLE
[Intermediate] - Change pokemon stats with radar chart

### ISSUE THREAD

**Issue by igihcksn** [igihcksn/mypocketmonsters#28]
Change stats section with radar chart, u can implement this [react-chartjs-2](https://react-chartjs-2.js.org/examples/radar-chart/) custom component base on chart js.
<img width="382" alt="Screen Shot 2022-09-14 at 14 26 21" src="https://user-images.githubusercontent.com/22608147/190088896-b9729581-6b64-45b7-9c40-6cc1a4458caa.png">

**Comment by yamitrvg12** [comment]
Hey [igihcksn](https://github.com/igihcksn), can you assign me this task so that I can work on it?

**Comment by igihcksn** [comment]
Yes sure @yamitrvg12 after u done with this https://github.com/igihcksn/mypocketmonsters/issues/14 issue i'll assign this issue to you
